tests/test_gitlab/events/pull_request.py

```diff
@@ -0,0 +1,361 @@
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+            {
+                "id": 40565195,
+                "title": "discussion",
+                "color": "#428bca",
+                "project_id": 69912300,
+                "created_at": "2025-05-16 15:09:39 UTC",
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+            {
+                "id": 40565192,
+                "title": "documentation",
+                "color": "#f0ad4e",
+                "project_id": 69912300,
+                "created_at": "2025-05-16 15:09:39 UTC",
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+            {
+                "id": 40565197,
+                "title": "enhancement",
+                "color": "#5cb85c",
+                "project_id": 69912300,
+                "created_at": "2025-05-16 15:09:39 UTC",
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+            {
+                "id": 40565196,
+                "title": "suggestion",
+                "color": "#428bca",
+                "project_id": 69912300,
+                "created_at": "2025-05-16 15:09:39 UTC",
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+            {
+                "id": 40565194,
+                "title": "support",
+                "color": "#f0ad4e",
+                "project_id": 69912300,
+                "created_at": "2025-05-16 15:09:39 UTC",
+                "updated_at": "2025-05-16 15:09:39 UTC",
+                "template": False,
+                "description": None,
+                "type": "ProjectLabel",
+                "group_id": None,
+            },
+        ],
+        "last_commit": {
+            "id": "7aa00b22e26de3efc8270aebc9ee51c40107d4ff",
+            "message": "Test commit\n\nSigned-off-by: Shounak Dey <shounakdey@ymail.com>\n",
+            "title": "Test commit",
+            "timestamp": "2025-05-16T21:24:13+05:30",
+            "url": "https://gitlab.com/sdglitched/w2fm-test/-/commit/7aa00b22e26de3efc8270aebc9ee51c40107d4ff",
+            "author": {"name": "Shounak Dey", "email": "shounakdey@ymail.com"},
```

Comment:
Please remove your email address and add some random placeholder here like `testaddr@testaddr.test`.